core/block_svg.js

```diff
@@ -272,18 +272,23 @@ Blockly.BlockSvg.prototype.getIcons = function() {
  * @param {Blockly.BlockSvg} newParent New parent block.
  */
 Blockly.BlockSvg.prototype.setParent = function(newParent) {
-  if (newParent == this.parentBlock_) {
+  if (newParent === this.parentBlock_) {
```

Comment:
@rachel-fenichel I think you mentioned at some point preferring double equals in this repo, is that right?

Reply:
Happy to change, I'm not going to pretend like the difference between `==` and `===` will save us on perf in the long run, especially since both of these are objects / null.  However if you look at the spec language for the `===` and `==` operator it's fairly clear that `null === object` is false much sooner than the `==` case.  I'm not sure why you would ever opt-into type-looseness here.

Reply:
I'm of the opinion that you should use the rules enforced by your linter/style guide with this particular argument.  I'll switch back to `==` here to be consistent.

Though it is worth noting that eslint doesn't actually have a rule that will ban threequals, if it did, I'd ask that we enable it as it would stop me from making this mistake again in the future :)